src/mount/readdata.h

```diff
@@ -211,11 +210,19 @@ struct ReadRecord {
 	std::atomic<uint8_t> refreshCounter = 0;
 	std::atomic<uint16_t> requestsNotDone = 0;
 	bool expired = false; //gMutex
+	std::atomic<bool> stopThread;
+	std::thread garbageCollectorThread;
```

Comment:
Looks like every ReadRecord now contain a garbage collection thread. I am not familiar with this part, but, does it means that if we are reading from 100 files in parallel, we will have 100 GC threads?
Creating and destroying threads is expensive.

Reply:
This was replaced. Now it is done in the read_delayed_ops thread.